syft/execution/action.py

```diff
@@ -1,7 +1,9 @@
 from abc import ABC
+from syft.interfaces.protobuf_interface import ProtobufInterface
+from syft.interfaces.msgpack_interface import MsgpackInterface
 
 
-class Action(ABC):
+class Action(ABC, ProtobufInterface, MsgpackInterface):
```

Comment:
For the sake of brevity, could we combine these interfaces and call the resulting class `SyftSerializable` or something like that?

Reply:
I have no idea why I haven't done this in the first place. Nice one.